### ISSUE TITLE
Cannot resolve bundle chunk-vendors

### ISSUE THREAD

**Issue by ghadjigeorghiou** [django-webpack/django-webpack-loader#237]
I don't really understand why I get this error and how to fix this.  As far as I know it is raised from the loader.py module from these lines:
```python
 if assets.get('status') == 'done':
       chunks = assets['chunks'].get(bundle_name, None)
       if chunks is None:
            raise WebpackBundleLookupError('Cannot resolve bundle {0}.'.format(bundle_name))
        return self.filter_chunks(chunks)
```

My settings.py file:
```python
VUE_FRONTEND_DIR = os.path.join(BASE_DIR, 'vue_frontend')

points django-webpack-loader to the stats file we generate during our Vue build
WEBPACK_LOADER = {
    'DEFAULT': {
        'CACHE': not DEBUG,
        'BUNDLE_DIR_NAME': 'vue/',  # must end with slash
        'STATS_FILE': os.path.join(VUE_FRONTEND_DIR, 'webpack-stats.json'),
        'POLL_INTERVAL': 0.1,
        'TIMEOUT': None,
        'IGNORE': [r'.+\.hot-update.js', r'.+\.map']
    }
}
```

My vue.config.js contains this:
```javascript
config.optimization
            .splitChunks({
                cacheGroups: {
                    vendor: {
                        test: /[\\/]node_modules[\\/]/,
                        name: "chunk-vendors",
                        chunks: "all",
                        priority: 1
                    },
                },
            });
```

**Comment by shonin** [comment]
Hi @ghadjigeorghiou, I have just released an alternative to this package that handles split chunks quite well. If you're still in need to solve this problem give it a shot and let me know how it goes: [Django Manifest Loader](https://github.com/shonin/django-manifest-loader)

**Comment by rvlb** [comment]
We've merged #232 which solves this issue when used with `webpack-bundle-tracker@1.0.0-alpha.1`. Release will follow soon and docs will also be updated.
